jenkins-builds/7.0.UP02-20.08-dom-mc

```diff
@@ -1,63 +1,64 @@
- MUST-v1.6-CrayGNU-20.08.eb                             --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP02/mc/easybuild/tools
- Scalasca-2.5-CrayGNU-20.08.eb                          --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP02/mc/easybuild/tools
- Score-P-6.0-CrayGNU-20.08.eb                           --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP02/mc/easybuild/tools
- scorep_binding_python-3.0-CrayGNU-20.08-python3.eb     --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP02/mc/easybuild/tools
- advisor_2020_update2.eb                                --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP02/mc/easybuild/tools
- ddt-20.0-Suse-15.0.eb                                  --installpath=$APPS/UES/jenkins/7.0.UP02/mc/easybuild/tools
- ddt-20.1.1-Suse-15.0.eb                                --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP02/mc/easybuild/tools
- cgdb-v0.7.1.eb                                         --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP02/mc/easybuild/tools
- ddd-3.3.12.eb                                          --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP02/mc/easybuild/tools
- gperftools-2.8.eb                                      --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP02/mc/easybuild/tools
- inspector_2020_update2.eb                              --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP02/mc/easybuild/tools
- likwid-1d6636c.eb                                      --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP02/mc/easybuild/tools
- malt-1.2.0-CrayGNU-20.08.eb                            --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP02/mc/easybuild/tools
- mpiP-57fc864-CrayGNU-20.08.eb                          --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP02/mc/easybuild/tools
- njmon-v64-SLES15.eb                                    --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP02/mc/easybuild/tools
- otf2_cli_profile-9be64de-CrayGNU-20.08.eb              --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP02/mc/easybuild/tools
- papi-wrap-57150c8-CrayGNU-20.08.eb                     --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP02/mc/easybuild/tools
- pprof-1a94d86.eb                                       --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP02/mc/easybuild/tools
- pudb-2019.2-CrayGNU-20.08-python3.eb                   --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP02/mc/easybuild/tools
- termgraph-0.4.2-python3.eb                             --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP02/mc/easybuild/tools
- vtune_profiler_2020_update2.eb                         --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP02/mc/easybuild/tools
+ Amber-20-3-0-CrayIntel-20.08.eb                        --set-default-module

```

Comment:
is this a conflict with https://github.com/eth-cscs/production/pull/1846 (unix format) ?

Reply:
Thanks for pointing that out @henrique: it should be fixed now.